src/components/elements/Notification/FloatNotification.tsx

```diff
@@ -38,36 +83,81 @@ const FloatNotification = ({ data }: FloatNotificationProps) => {
-                  <div className="mt-2 flex items-center gap-2">
-                    <LinearProgressBar value={parseInt(item.value)} className="h-2 bg-success-40" color="success-600" />
-                    <Text variant="text-12-semibold" className="text-black">
-                      {item.value}
+              {isLoaded &&
+                notAcknowledgedJobs &&
+                notAcknowledgedJobs.map((item, index) => (
+                  <div key={index} className="rounded-lg border-2 border-grey-350 bg-white p-4 hover:border-primary">
+                    <div className="mb-2 flex items-center gap-1">
+                      <div className="h-2 w-2 rounded-full bg-primary" />
+                      <Text variant="text-14-light" className="leading-[normal] text-darkCustom " as={"span"}>
+                        {item.name}
+                      </Text>
+                    </div>
+                    <Text variant="text-14-light" className="text-darkCustom">
+                      Site: <b>{item.entityName}</b>
                     </Text>
+                    <div className="mt-2">
+                      {item.status === "failed" ? (
+                        <Text variant="text-12-semibold" className="text-error-600">
+                          {item.payload ? getErrorMessageFromPayload(item.payload) : "Failed to complete"}
+                        </Text>
+                      ) : (
+                        <div className="flex items-center gap-2">
+                          {item.name === "Polygon Upload" &&
+                          (item.processedContent === null || item.totalContent === null) &&
+                          item.status === "pending" ? (
+                            <div style={{ width: "100%" }}>
+                              <LinearProgress
+                                sx={{
+                                  height: 9,
+                                  borderRadius: 99,
+                                  backgroundColor: "#a9e7d6",
+                                  "& .MuiLinearProgress-bar": { backgroundColor: "#29c499" }
+                                }}
+                              />
+                            </div>
+                          ) : (
+                            <LinearProgressBar
+                              value={
+                                item.status === "succeeded"
+                                  ? 100
+                                  : ((item.processedContent ?? 0) / (item.totalContent ?? 1)) * 100
+                              }
+                              className="h-2 bg-success-40"
+                              color="success-600"
+                            />
+                          )}
+                          <Text variant="text-12-semibold" className="text-black">
+                            {item.name === "Polygon Upload"
+                              ? item.status === "succeeded"
+                                ? "Done!"
```

Comment:
Please make sure any text that is displayed gets wrapped in `t()`

Reply:
added to every place but names